gui_qt/project_state.py

```
@@ -170,15 +192,20 @@ def _load_game_root_from_config(self) -> None:
             game_root = data.get("game_root")
             if isinstance(game_root, str) and game_root.strip():
-                self._game_root = self._path_without_resolve(game_root)
+                original = self._path_without_resolve(game_root)
+                effective, adjusted = self.normalize_game_root(game_root)
+                self._game_root = effective
+                if adjusted:
+                    self._game_root_redirect_from = original
+                    self._save_game_root_to_config(effective)
         except Exception:
             # Non-fatal for GUI
             pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)